react-loops.d.ts

```diff
@@ -45,3 +45,16 @@ export function ElseIf(props: {
 }): React.ReactNode;
 
 export function Else(props: { children: React.ReactNode }): React.ReactNode;
+
+export function Choose(props: {
+  children: When | Otherwise;
```

Comment:
I'm not a typescript wizard yet so I don't know what this should be, but I think you want to allow an ReactElement or Array of ReactElement of `When | Otherwise` type